### ISSUE TITLE
[Question] Facing a weird issue while creating a ticket

### ISSUE THREAD

**Issue by wolfassi123** [facetoe/zenpy#603]
So I have been testing aroud zenpy and faced a weird issue that I wanted to make sure of.
So I am trying to create a User, then create a ticket and finally reply as an agent (adding comment).

I am using the following bit of code:
```
from zenpy.lib.api_objects import Ticket, User

costumer = User(name='Johny', email='Johyn@example.com',id=111)
agent_id = 14538682961565

test_ticket = Ticket(description='Issue with my latest order please help me',
           requester=costumer,
           assignee_id=agent_id
                     )

costumer_id = costumer.id

zenpy_client.tickets.create(test_ticket)
test_ticket.comment = Comment(author_id = costumer_id,body="Hello there", public=True)
zenpy_client.tickets.update(test_ticket)
```

So when I use this, the ticket is not being assigned an "Automatic" ID. It is stating that its ID is simply None `"Ticket(id=None)"`

So I decided to manually add the id and got the following bit of code:
```
from zenpy.lib.api_objects import Ticket, User

costumer = User(name='Johny', email='Johyn@example.com',id=111)
agent_id = 14538682961565

test_ticket = Ticket(description='Issue with my latest order please help me',
           requester=costumer,
           assignee_id=agent_id,
                     id=75
                     )

costumer_id = costumer.id

zenpy_client.tickets.create(test_ticket)
test_ticket.comment = Comment(author_id = costumer_id,body="Hello there", public=True)
zenpy_client.tickets.update(test_ticket)
```
Here the ticket is created as I am being return `"Ticket(id=75)"` when I print it out. But the problem is that I am getting the following error : 
```
APIException: {"error": {"title": "Attribut non valide", "message": "Valeur non valide pour l\u2019attribut id. Invalid parameter: id must be an integer from api/v2/tickets/update"}}
```

Now I noticed that the only way to solve this error is if the ticket ID is exactly the previously created id, but +1.
To further elaborate, on Zendesk (the web app) each order has a certain ***"#Order"*** next to it. Each time I create a ticket, it has to be exactly the next order number or else I won't be able to update the ticket or anything (To note that the ticket is created, but has a different ID on zendesk than how I passed the argument, which causes the issue whenever I call the ".update" and the ticket is hence not updated, simply created under a different ID than the one that I passed".

Is there something that I am missing? Am I doing something wrong?
Basically I am trying to simulate a chat between a user and an agent as I am trying to implement a chatbot to help out on some issues on Zendesk, so production wise, I need to be able to dynamically create different instances of the Class "User" or "Ticket".

**Comment by cryptomail** [comment]
I will look at this...sorry for the delay :)

**Comment by cryptomail** [comment]
Regarde ca.... 
Like create, the update method returns the modified object.
http://docs.facetoe.com.au/zenpy.html#usage
